modules/microshift-olm-deploy-ops-global-ns.adoc

```diff
@@ -0,0 +1,119 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-OLM-deploy-Operators_{context}"]
+= Adding OLM-based Operators to a networked cluster using the global namespace
+
+Use this procedure if you want to deploy different operators to different namespaces. For {microshift-short} clusters that have network connectivity, Operator Lifecycle Manager (OLM) can access sources hosted on remote registries. The following procedure lists the basic steps of using configuration files to install an Operator that uses the global namespace.
+
+.Prerequisites
+* The {oc-first} is installed.
+* Operator Lifecycle Manager (OLM) is installed.
+* You have created a custom catalog in the global namespace.
+
+.Procedure
+
+. Confirm that OLM is running by using the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-operator-lifecycle-manager get pod -l app=olm-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME                            READY   STATUS    RESTARTS   AGE
+olm-operator-66c97f894f-55wp2   1/1     Running   0          126m
+----
+
+. Confirm that the pod the catalog runs in is active by using the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-operator-lifecycle-manager get pod -l app=catalog-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                READY   STATUS    RESTARTS   AGE
+catalog-operator-7cf77c459d-jbrd7   1/1     Running   0          126m
+----
+
+//Q which directory do we place these YAMLs in? do we need to specify?
+[NOTE]
+====
+The following steps assume you are using the global namespace, `openshift-marketplace`. The catalog must run in the same namespace as the Operator. The Operator must support the *All namespaces* mode.
+====
+
+. Create the `CatalogSource` object by using the following command and example YAML:
++
+[source,terminal]
+----
+$ sudo cat _<my-catalog-source.yaml>_ <1>
```

Comment:
I think `sudo` is not necessary. `oc` doesn't need to be run as privileged, but if  the file is root-owned then `oc` would also need `sudo`